lib/chat_web/live/main_live/index.ex

```diff
@@ -34,13 +34,15 @@ defmodule ChatWeb.MainLive.Index do
       if action == :export do
         socket
         |> assign(:need_login, false)
+        |> assign(:keys_stored, true)
```

Comment:
Wording is hard here
- keys tried
- keys secured
- keys retrieved
- handshaked